notes/stories/film/film-scifi-2000s.st.txt

```diff
@@ -3674,6 +3674,52 @@ father and daughter [Dr. Shinichi Chujo's adult daughter informed him that his g
 giant moth [Mothra came to Japan's defense when Godzilla attacked.]
 
 
+movie: Paycheck (2003)
+======================
+
+:: Title
+Paycheck
+
+:: Description
+Paycheck is a 2003 American science fiction action film based on the 1953
+short story of the same name by science fiction writer Philip K. Dick.
+
+Synopsis: A talented engineer accepts a contract to work on a three year
+secret project after which he will have his memory of the time erased and will
+be made rich for his efforts.
+
+:: Date
+2003-12-25
+
+:: References
+https://en.wikipedia.org/wiki/Paycheck_(film)
+
+:: Ratings
+3 <paul>
+
+:: Choice Themes
+speculative prediction of the future [A main novelty of the film was a special machine that allowed people to view the future.]
+
+:: Major Themes
+facing organized pursuit [Jennings was being pursued by both the FBI and Allcom corporation goons.]
+memory erasure [Central to the plot was Jennings having his memories of the last three years erased from his mind.]
+unethical business practices [The Allcom corporation made use of classified government information to illegally build a machine to see into the future. An unnamed corporation hired Jennings to reverse engineer a competitor's technology and recreate it with improvements.]
+intellectual property in society [The viewer is made to ponder the extent to which corporations should be allowed to go to protect their intellectual property. In particular, is the practice of contractually obligating people to undergo mindwipes in the name of protecting intellectual property justified?]
```

Comment:
Was this question really on the table or are you going out on a limb here? From reading the synopsis it sounds like a standard NDA kind of thing with a futuristic twist. Trade secrets and intellectual property aren't exactly the same thing.

Reply:
Hum.. let's consider what we should do here.

It seems to me there are two questions: 1) Does [intellectual property in society](https://en.wikipedia.org/wiki/Intellectual_property) apply? and 2) to what extent was it featured?

As for Q1: The crux is that NDAs have been replaced with special contracts whereby people work for some pre-specified time on a project, and then have their memories of the time period in question wiped to ensure they can't tell anyone about it.

Another consideration is that the [intellectual property in society](https://en.wikipedia.org/wiki/Intellectual_property) theme covers trade secrets as a special case; the wikipedia reference also includes trade secrets as a type of intellectual property.

So I think it fair to apply it here.

As for Q2: I suppose **major** is a bit much as it was really only entertained toward the start of the film in a particular scene involving the protagonist and the corporate CEO guy.

Conclusion: Keep but knock down to minor.

Thoughts?

Reply:
> It doesn’t sound to me like this story is interested in discussing any
legal questions about property. Nor does your comments make clear that
there was any legally recognised (intellectual) property (I should not have
used the term trade secret).

> It sounds like this is a story about a company that has so much clout that
it can unethically bully people into doing horrible things. Like “cut off
your arm if you wanna continue working here”. We have something for that, I
recall, or we can make a new theme if we need something more specific.

> M

I see. Well, in that case, we could coin a new **onerous non-disclosure agreements** to go along with the [onerous license agreements](https://themeontology.org/theme.php?name=onerous%20license%20agreements) theme and use the motivation: "The story explores a near-future where corporations make use of mind wipe including non-disclosure agreements. Two different corporations signed the protagonist to such contracts, suggesting that these sorts of agreements were somehow commonplace."

Thoughts?

Reply:
> It doesn’t sound to me like this story is interested in discussing any
legal questions about property. Nor does your comments make clear that
there was any legally recognised (intellectual) property (I should not have
used the term trade secret).

> It sounds like this is a story about a company that has so much clout that
it can unethically bully people into doing horrible things. Like “cut off
your arm if you wanna continue working here”. We have something for that, I
recall, or we can make a new theme if we need something more specific.

> M

Updated in light of yesterday's discussion.